helper/schema/core_schema_test.go

```diff
@@ -34,15 +34,6 @@ func testResource(block *configschema.Block) *configschema.Block {
 }
 
 func TestSchemaMapCoreConfigSchema(t *testing.T) {
-	// these are global so if new tests are written we should probably employ a mutex
-	DescriptionKind = StringMarkdown
-	SchemaDescriptionBuilder = func(s *Schema) string {
-		if s.Required && s.Description != "" {
-			return fmt.Sprintf("**Required** %s", s.Description)
-		}
-		return s.Description
-	}
```

Comment:
This is technically reducing the coverage of this test, but it's so specific that I don't think we're losing too much 😆